swatch-system-conduit/src/test/java/org/candlepin/subscriptions/conduit/InventoryControllerTest.java

```diff
@@ -946,4 +946,26 @@ void testSystemMemoryBytesOverMaxValueSetsToNull() throws ApiException {
     verify(inventoryService).scheduleHostUpdate(cfacts);
     verify(inventoryService, times(1)).flushHostUpdates();
   }
+
+  @Test
+  void testNumberOfCpusIsMappedToConduitFacts() {
+    int expectedNumberOfCpus = 5;
```

Comment:
Being a little picky here, but why not make expectedNumberOfCpusa String to begin with? This would allow you to use the value directly when setting the fact, instead of appending the "" when setting the value to make it a string  (same with expectedThreadsPerCore below).